test/ctia/test_helpers/http.clj

```diff
@@ -42,7 +42,9 @@
               :body new-entity
               :headers {"api_key" api-key})]
     (when (not= 200 status)
-      (throw (ex-info (str "Expected status to be 200 but was %s" status)
+      (throw (ex-info (str "Expected status to be 200 but was " status
+                           " for " path ":\n"
+                           (with-out-str (clojure.pprint/pprint response)))
```

Comment:
@yogsototh if you're just going to string-concat, then why use `ex-info` at all. You could just use `(Exception. (str a b c))`